omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/DrugResource1_10.java

```diff
@@ -52,8 +60,44 @@ public DelegatingResourceDescription getCreatableProperties() {
 		DelegatingResourceDescription description = super.getCreatableProperties();
 		description.addProperty("strength");
 		description.addRequiredProperty("name");
+		description.addProperty("drugReferenceMaps");
 		
 		return description;
 	}
 	
+	public Model getGETModel(Representation rep) {
+		ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);
+		if (rep instanceof DefaultRepresentation) {
+			modelImpl
+			        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapGetRef"));
+		} else if (rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapGet"));
+		}
+		return modelImpl;
+	}
+	
+	@Override
+	public Model getCREATEModel(Representation rep) {
+		return new ModelImpl()
+		        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapCreate"));
+	}
+	
+	@PropertySetter("drugReferenceMaps")
+	public static void setGroupMembers(Drug drug, List<DrugReferenceMap> members) {
+		for (DrugReferenceMap member : members) {
```

Comment:
It would be better to have a if control statement to check if "members=!NULL && members.size()>0 "before entering the for loop.

Reply:
your right @vinayvenu

Reply:
hey! I'm Yuvaneeth..not vinay venu!!

Reply:
i have added the check statement @Yuvaneeth